scripts/configure_test_suite_network.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+NETWORK_STRING=$1
+
+PATH_TO_SCRIPT=$(realpath $0)
+PATH_TO_SCRIPTS_DIRECTORY=$(dirname $PATH_TO_SCRIPT)
+PATH_TO_PROJECT_ROOT=$(dirname $PATH_TO_SCRIPTS_DIRECTORY)
+PATH_TO_PACKAGES="${PATH_TO_PROJECT_ROOT}/packages"
+# 1 level above monorepo root
+PATH_TO_ROOT=$(dirname $PATH_TO_PROJECT_ROOT)
+
+# use path from the argument ($2), otherwise default to the ../dash-network-configs
+DASH_NETWORK_CONFIGS_PATH="${2:-"$(realpath "$PATH_TO_ROOT/dash-network-configs")"}"
+
```

Comment:
Let's check that yq is install. If not please print an error and provide a link to https://github.com/mikefarah/yq/#install